src/encoded/docs/adding_new_assays.md

```diff
@@ -0,0 +1,156 @@
+
+        * Add 1 to the numbered results in [tests/features/views.feature]:
+        ```
+        When I click the link to "/matrix/?type=Experiment"
+        Then I should see "Showing 72 results"
+        ```
+        * Add 1 to the ```len(lines)``` in [tests/test_batch_download.py]:
+        ```
+        def test_batch_download_report_download(testapp, index_workbook):
+        ...
+        assert len(lines) == 72
+        ```
+        * Assess if your new insert will break any other tests- the easiest way to do this is simply to run the tests and interpret any errors that come up. The tests are listed in sets in the [encoded README.md].
+
+    * Add a new audit test. To ensure assays with the new assay term name will be audited correctly, you need to create or edit a fixture (a sample experiment object, similar to your insert) and test that it is audited the same way as experiments of other assay types. For example, if you were to test that your new **CUT&Tag** term was added correctly to ```targetRequiredAssayList```, you can add new lines to the existing test for this audit.
+
+        * In [tests/test_audit_experiment.py], you'll find this test for checking targets are included in experiments that require them:
+
+        ```
+        def test_audit_experiment_target(testapp, base_experiment):
+            testapp.patch_json(base_experiment['@id'], {'assay_term_name': 'ChIP-seq'})
+            res = testapp.get(base_experiment['@id'] + '@@index-data')
+            assert any(error['category'] == 'missing target'
+                for error in collect_audit_errors(res))
+        ```
+
+        * Then, you can patch a fixture (```base_experiment``` is imported in the test above) already used in the audit test to check your new assay term, for example:
+
+        ```
+        testapp.patch_json(base_experiment['@id'], {'assay_term_name': 'CUT&Tag'})
+            res = testapp.get(base_experiment['@id'] + '@@index-data')
+            assert any(error['category'] == 'missing target'
+                for error in collect_audit_errors(res))
+        ```
+        * If you prefer, you can also write a new experiment fixture which you would add to [tests/fixtures/schemas/experiment.py]. For more detailed instructions, please refer to [making_audits.md].
+
+4. Ontology: 
+
+    Check to see if the new assay type has an entry in [OBI].
+    * If there is an entry, you will use the OBI identifier (e.g., OBI:0002039).
+    * If there is no entry, write a New Term Request ticket on the [NTR board], and provide a publication describing the assay type. The ticket number will be used as your assay term identifier (e.g., NTR-564 becomes NTR:0000564).
+
+    Add the assay term name and identifier to [types/assay_data.py].
+
+    If you using an NTR identifier, add an entry to [src/encoded/commands/ntr_terms.py]. You will need to provide:
+    * The name for your assay type in the ```name``` property
+    * A shortened form of the name in ```preferred_name```
```

Comment:
I think clarification here would be good. You could say something about how the name is usually the non-abbreviated name of the assay. What you have as preferred_name should match what you would like added as the assay_term_name in all of the schemas.